waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -202,6 +217,43 @@ public DatabaseMapping databaseMapping(@NotNull String databaseName) throws NoSu
         "Waggle Dance error no database mapping available tried to map database '" + databaseName + "'");
   }
 
+  @Override
+  public void checkTable(String databaseName, String tableName, DatabaseMapping mapping)
```

Comment:
I wonder if `isTableAllowed` would be a better name for this? `check` feels a bit vague to me.

Reply:
or `checkTableAllowed` since it doesn't return a `boolean`.

Reply:
Not really, cause it only throws an error, it doesn't return any boolean.

Reply:
ah right, the second comment appeared later 😄